src/WidgetBase.ts

```diff
@@ -198,13 +191,14 @@ export class WidgetBase<P = WidgetProperties, C extends DNode = DNode> extends E
 		this._properties = <P> {};
```

Comment:
I think generally it does, I don't think it'll reflect in the benchmarks but I'll update it regardless.